services/web/server/src/simcore_service_webserver/projects/projects_exceptions.py

```diff
@@ -42,3 +43,8 @@ def __init__(self, project_uuid: str, node_uuid: str):
         super().__init__(f"Node {node_uuid} not found in project {project_uuid}")
         self.node_uuid = node_uuid
         self.project_uuid = project_uuid
+
+
+class HTTPLocked(web.HTTPClientError):
+    # pylint: disable=too-many-ancestors
+    status_code = 423

```

Comment:
here I am not sure I agree.

the HTTPLocked exception is not something inherent to projects but to REST servers. so I think it was better in projects_handlers.py. It's actually why I put it there. This HTTP code is missing from aiohttp.

Reply:
OK. My intention here was breaking coupling w/ where this class was before. 
Since now i see it is missing from aiohttp, i will then add it in ``servicelib.aiotthp.web_exceptions_extra.py``